### PR TITLE
dev-util/geany-plugins: version bump to 1.33

### PR DESCRIPTION
**Pull request by cjmayo**
Package-Manager: Portage-2.3.19, Repoman-2.3.6

-----

Basically a rename, geanylatex plugin is renamed to latex (and keyrecord was out of alphabetical order).

```
--- geany-plugins-1.32.ebuild
+++ geany-plugins-1.33.ebuild
@@ -82,3 +82,2 @@
 		--enable-geanyinsertnum
-		--enable-geanylatex
 		--enable-geanymacro
@@ -88,5 +87,6 @@
 		--enable-geanyvc $(use_enable gtkspell)
+		--enable-keyrecord
+		--enable-latex
 		--enable-lineoperations
 		--enable-lipsum
-		--enable-keyrecord
 		--enable-overview
chris@dvorak /srv/open/github/gento
```

### CODE REVIEW

File: dev-util/geany-plugins/geany-plugins-1.33.ebuild [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+		--disable-webhelper
+	)
+
+	econf "${myeconfargs[@]}"
+}
+
+src_install() {
+	default
+
+	prune_libtool_files --modules
```
Comment (mgorny) [inline dev-util/geany-plugins/geany-plugins-1.33.ebuild]: Are there any `.la` files left? If not, then please do instead:

    find "${D}" -name '*.la' -delete || die

(and drop eutils)

File: dev-util/geany-plugins/geany-plugins-1.33.ebuild [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+	# make installs all translations if LINGUAS is empty
+	if [[ -n "${LINGUAS+x}" && -z "$LINGUAS" ]]; then
```
Comment (mgorny) [inline dev-util/geany-plugins/geany-plugins-1.33.ebuild]: How about `[[ -z ${LINGUAS-x} ]]`?

File: dev-util/geany-plugins/geany-plugins-1.33.ebuild [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+		rm -r "${D}/usr/share/locale/" || die
```
Comment (mgorny) [inline dev-util/geany-plugins/geany-plugins-1.33.ebuild]: `"${ED%/}"`

File: dev-util/geany-plugins/geany-plugins-1.33.ebuild (hunk 1 above)
Comment (cjmayo) [reply dev-util/geany-plugins/geany-plugins-1.33.ebuild]: Yes .la files still created. Replaced with find and removed eutils.

File: dev-util/geany-plugins/geany-plugins-1.33.ebuild (hunk 2 above)
Comment (cjmayo) [reply dev-util/geany-plugins/geany-plugins-1.33.ebuild]: That's neater. Done.

File: dev-util/geany-plugins/geany-plugins-1.33.ebuild (hunk 3 above)
Comment (cjmayo) [reply dev-util/geany-plugins/geany-plugins-1.33.ebuild]: OK.

**Comment by cjmayo** [comment]
Thanks for the review.

**Pull request closed without merging**